decko-rails/lib/decko/engine.rb

```diff
@@ -44,9 +44,16 @@ class Decko::Engine < Rails::Engine
 
     initializer :connect_on_load do
       ActiveSupport.on_load(:active_record) do
-        Cardio.add_card_paths Decko::Engine.paths, Rails.application.config, Wagn.root
+        Cardio.card_config Rails.application.config, Decko::Engine.paths, Wagn.root, Rails.cache
         ActiveRecord::Base.establish_connection(Rails.env)
       end
+      ActiveSupport.on_load(:after_initialize) do
+        begin
+          require_dependency 'card' unless defined?(Card)
+        rescue ActiveRecord::StatementInvalid => e
```

Comment:
Console wasn't loading Card class right, so I added this require.  But in some of the rake tasks this throws errors because there is no table.  Seemed easier to just eat the error than to figure out how to load it sometimes, but not others.